app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import datetime
+import time
+
+
+def create_file_every_second():
+    while True:
+        file_name = 'app-' + datetime.datetime.now().strftime("%-H_%-M_%-S") + '.log'
```

Comment:
It's better to store `datetime.datetime.now()` to some variable and reuse it in line 9
`time.strftime("%-H_%-M_%-S")` is equal to str(time)